src/main/java/core/basesyntax/service/TransactionParser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.TransactionDto;
+import java.util.List;
+
+public interface TransactionParser {
+    List<TransactionDto> parserTransactionOperation(List<String> line);
```

Comment:
```suggestion
    List<TransactionDto> parse(List<String> line);
```
I think @karma-o meant this